packages/trigger/src/index.ts

```diff
@@ -77,11 +82,12 @@ export class TriggerCache<T> {
  * Trigger signals added to the cache only when tracked under a computation,
  * and get deleted from the cache when they are no longer tracked.
  *
- * @returns a tuple of `[track, dirty]` functions
+ * @returns an array of `[track, dirty, dirtyAll]` functions
```

Comment:
3 elements still make a tuple :)
```suggestion
 * @returns a tuple of `[track, dirty, dirtyAll]` functions
```